company_linkage/parat_scripts/get_ai_counts.py

```diff
@@ -160,50 +162,57 @@ def run_query_id_papers(self, table_name: str, test: bool = False) -> list:
         :param test: False if not running as a unit test
         :return:
         """
-        companies_query = f"""SELECT CSET_id, ror_id FROM 
-        `gcp-cset-projects.high_resolution_entities.aggregated_organizations`"""
-        if test:
-            companies_query += """ LIMIT 25"""
+        companies_query = f"""-- SELECT CSET_id, ror_id FROM 
+        # `gcp-cset-projects.high_resolution_entities.aggregated_organizations`"""
+        # if test:
+        #     companies_query += """ LIMIT 25"""
         client = bigquery.Client()
-        query_job = client.query(companies_query)
+        # query_job = client.query(companies_query)
         company_rows = []
-        for i, row in enumerate(query_job):
-            self.company_ids.append(row["CSET_id"])
-            if row["CSET_id"] not in self.regex_dict:
+        for i, cset_id in enumerate(self.cset_ids):
+            if test and i == 25:
+                break
+            # self.company_ids.append(cset_id)
+            if cset_id not in self.regex_dict:
                 # if it's not in the regex_dict that's bad
-                print(row["CSET_id"])
+                print(cset_id)
             else:
-                regexes = self.regex_dict[row['CSET_id']]
+                regexes = self.regex_dict[cset_id]
                 query = f"""SELECT DISTINCT merged_id, year, cv, nlp, robotics FROM `{table_name}`
                          WHERE regexp_contains(org_name, r'(?i){regexes[0]}') """
                 # if we have more than one regex for an org, include all of them
                 if len(regexes) > 1:
                     for regex in regexes[1:]:
                         query += f"""OR regexp_contains(org_name, r'(?i){regex}') """
-                if row["ror_id"]:
-                    self.ror_dict[row["CSET_id"]] = row["ror_id"]
-                    query += f"""OR ror_id IN ({str(row["ror_id"])[1:-1]})"""
+                if cset_id in self.ror_dict:
+                    # self.ror_dict[row["CSET_id"]] = row["ror_id"]
```

Comment:
Couple more commented out lines of what looks like it could be substantive code - just noting it in case there was a mistaken commit with test conditions or something. No worries if you intended to comment this stuff out!

Reply:
Nope, this was an old way of doing things I commented out as I switched over to a new method and was testing. I'll delete. Thanks!